extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/MSQWorkerTaskLauncher.java

```diff
@@ -390,6 +391,12 @@ public long getDuration()
     {
       return duration;
     }
+
+    @JsonProperty("pendingMs")
+    public long getPendingTimeInMS()
```

Comment:
`pendingTimeMillis` would probably be a better name and more in line with rest of the names in Druid.

Reply:
Since we already have `durationMs`  as part of line:389 of this payload, have `pendingTimeMillis` would be weird. If you feel strongly, I can change.

Reply:
We are using millis as convention in this class, so it's a bit unfortunate that we already have "durationMs" in the taskReport. Would the best thing be to call it `pendingTimeMillis` in the code and expose it in the report as `getPendingTimeInMs`. As a nit, I think it should be `getPendingTimeInMs` though.